h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> We kicked ourselves the moment we allowed ourselves to implement this initially with a solution which was not complete and without a sufficiently motivating use case, user request, or any initial research.

I can't comment on our initial decision to implement and merge this feature, since I was not yet part of the team at that time. However, during the last two years, I don't remember hearing any feedback proving the allegation that the current feature is incomplete or half-baked. Are you aware of such feedback? If yes, could you please share it with me?

On the other hand, I thought (and still think) that there is a clear use-case for the user wanting to find an annotation inside the document, starting from the annotation card. Do you see a problem with that use-case?